central/credentialexpiry/service/service_impl.go

```diff
@@ -153,14 +162,82 @@ func (s *serviceImpl) getScannerCertExpiry(ctx context.Context) (*v1.GetCertExpi
 			errorList.AddError(err)
 			// All the endpoints have failed.
 			if len(errorList.ErrorStrings()) == len(clairifyEndpoints) {
-				return nil, errors.New(errorList.String())
+				return nil, errorList.ToError()
 			}
 		case expiry := <-expiryC:
 			return &v1.GetCertExpiry_Response{Expiry: expiry}, nil
 		}
 	}
 }
 
+func (s *serviceImpl) getScannerV4CertExpiry(ctx context.Context) (*v1.GetCertExpiry_Response, error) {
+	if !features.ScannerV4.Enabled() {
+		return nil, errors.Wrap(errox.InvalidArgs, "Scanner V4 is not integrated")
```

Comment:
The user should never see this error because the UI will not be showing the V4 section unless the feature is enabled. Since the install / upgrade should enable V4 by default, seems will only run into this if V4 is disabled after the fact - which implies some knowledge about Scanner V4.  I changed the error to `Scanner V4 is not enabled/integrated` to cover both scenarios mentioned.